bot/util/utilities.py

```diff
@@ -86,6 +90,72 @@ async def about(self, ctx):
         embed.add_field(name="Links:", value=str_links)
         await ctx.send(embed=embed)
 
+    @commands.command(name='embed')
+    async def embed(self, ctx, channel: str, color: str, *, text: str):
+        """Command Handler for the embed command
+
+        Creates and sends an embed in the specified channel with color and title and text. Title and text are separated
+        by a '|' character.
+
+        Args:
+            ctx (Context): The context in which the command was called.
+            channel (str): The channel where to post the message. Can be channel name (starting with #) or channel id.
+            color (str): Color code for the color of the strip.
+            text (str): The text to be posted in the embed. The string contains title and content, which are separated
+                        by a '|' character. If this character is not found, no title will be assumed.
+        """
+        try:
+            channel_to_post = get_text_channel(channel, ctx.guild)
+        except ValueError as error:
+            await ctx.send(error)
+            return
+
+        if '|' in text:
+            title, description = text.split('|')
+        else:
+            title = ''
+            description = text
+
+        try:
+            col = discord.Color(int('0x' + color, 16))
```

Comment:
bc I can only receive it as string and using that directly does not work. Took again carlbot commands as example

Reply:
But I'm doing exactly that in **utilities.py** with:
`embed = discord.Embed(title=ctx.guild.name, timestamp=datetime.utcnow(), color=constants.EMBED_COLOR_INFO)`
where the value of the constant is `0xFFDF00`.

Reply:
but CONSTANTS.EMBED_COLOR_INFO is a hex value (implicit casted to an int) , not a string. I could try using only the int conversion without the Color object, but only using the string does not work

Reply:
I know and that's what I meant.

Reply:
The advantage here is however, that if an invalid hex value is entered (an int that is not a valid color) then it will fail here already as opposed to when trying to send it